src/cmd/manage_nsfs.js

```diff
@@ -319,11 +336,12 @@ function set_access_keys(access_key, secret_key, generate) {
 // in name and new_name we allow type number, hence convert it to string
 async function fetch_account_data(action, user_input) {
     const { access_keys = [], new_access_key = undefined } = user_input.anonymous ? {} : get_access_keys(action, user_input);
+    const name = user_input.iam_name ?? user_input.name;
```

Comment:
I think it will create confusion, and it seems like now we will be allowed to create IAM accounts using the CLI (we didn't have it before), we needed this flag to access it for update/status/delete because we cannot identify account by its name only after the config change.

Reply:
I'm sorry, I don't understand what is the issue.
What do you mean by "config change"?

Generally, if the user specified an iam_name, we're using it.
Otherwise, it's a root account and there's only one "name". (You can say iam_name is the same as name).
I've added a comment that says the same.